features/image-transformations/overlay-using-layers.md

```diff
@@ -109,7 +109,8 @@ ImageKit supports many [image transformation parameters](../image-transformation
 | [ar](../image-transformations/resize-crop-and-other-transformations.md#aspect-ratio-ar)                        | Aspect ratio of the overlay image. It can also accept arithmetic expressions such as `iar_div_2`, or `bar`. Learn more about arithmetic expressions [here](../arithmetic-expressions-in-transformations.md). |
 | [c](../image-transformations/resize-crop-and-other-transformations.md#crop-crop-modes-and-focus)               | Cropping method. Accepts `force`, `at_max`, and `at_least`. |
 | [cm](../image-transformations/resize-crop-and-other-transformations.md#crop-crop-modes-and-focus)              | Crop mode. Supports `extract` and `pad_resize`. |
-| [fo](../image-transformations/resize-crop-and-other-transformations.md#focus-fo)                               | Relative focus area used during cropping. Accepts `face`, `custom`, `center`, `top`, `left`, `bottom`, `right`, `top_left`, `top_right`, `bottom_left`, and `bottom_right`. |
+| [fo](../image-transformations/resize-crop-and-other-transformations.md#focus-fo)                               | Relative focus area used during cropping. Accepts `face`, `custom`, `center`, `top`, `left`, `bottom`, `right`, `top_left`, `top_right`, `bottom_left`, and `bottom_right`. It also supports [object aware cropping](../image-transformations/resize-crop-and-other-transformations.md#object-aware-cropping). |
```

Comment:
```suggestion
| [fo](../image-transformations/resize-crop-and-other-transformations.md#focus-fo)                               | Relative focus area used during cropping. Accepts `face`, `custom`, `center`, `top`, `left`, `bottom`, `right`, `top_left`, `top_right`, `bottom_left`, and `bottom_right`. It also supports [object-aware cropping](../image-transformations/resize-crop-and-other-transformations.md#object-aware-cropping). |
```

Reply:
Yes i did for showing purpose replacing all images with demo , I have just got it